engines/router/missionctl/log/resultlog/fluentd.go

```diff
@@ -17,15 +17,11 @@ type fluentdClient interface {
 // fluentd backend.
 type FluentdLogger struct {
 	tag          string
-	bqLogger     BigQueryLogger
```

Comment:
Previously, this dependency existed so that the BQ logger (for streaming API) could be used to format the message in the same way for the batch logging with Fluentd. This is not important anymore since we're using the common `TuringResultLogEntry` methods.